week-2/debug/1.js

```diff
@@ -1,8 +1,33 @@
 // Predict and explain first...
+//Ans: The code defines a function called sum that takes two parameters a and b.
+
+//Inside the function, it simply prints out the result of adding a and b using return a * b.
```

Comment:
Be careful of `a * b` which multiplies. Here we are adding, so need a different operator